src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class Ball {
```

Comment:
its a good practice to implicitly specify access modifiers
```suggestion
public class Ball {
```